stats/ncstats.py

```diff
@@ -30,7 +30,7 @@ class NcStats(object):
     AVAILABLE_STRUCTURE_PARAMETERS = [
         'q_cumulative_duration', 'q_end', 'tot_vol_positive',
         'tot_vol_negative', 'time_q_max']
-    AVAILABLE_MANHOLE_PARAMETERS = ['s1_end', 'wos_duration']
+    AVAILABLE_MANHOLE_PARAMETERS = ['s1_end', 's1_max', 'wos_duration']
```

Comment:
It's better to add this to `NcStatsAgg.AVAILABLE_MANHOLE_PARAMETERS` (https://github.com/nens/threedi-qgis-plugin/blob/3a30fa3a7b92177ec193b0d75e2c9a454c2a83d6/stats/ncstats.py#L201) instead, because it's already available in the aggregation netCDF, and we probably only want it there and not in the regular netCDF.